Microsoft.Toolkit.Uwp.UI.Controls/MasterDetailsView/MasterDetailsView.cs

```diff
@@ -325,13 +325,13 @@ private void SetVisualState(VisualState state, bool animate)
 
         private void SetDetailsContent()
         {
-            if ((SelectedItem != null) && (_detailsPresenter != null))
+            if (_detailsPresenter != null)
             {
-                _detailsPresenter.Content = MapDetails == null
-                    ? SelectedItem
-                    : MapDetails(SelectedItem);
+               _detailsPresenter.Content = MapDetails == null
```

Comment:
Sorry to seem like a jerk, but please make sure the file is formatted properly. These new changes have only three spaces for a tab rather than four.